src/release/breaking-changes/routesettings-copywith-migration.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Migration guide for RouteSettings copyWith
+description: Removal of RouteSettings copyWith and how to migrate
+---
+
+## Summary
+
+RouteSettings copyWith was removed, and existing apps needed to use the constructor to create a new RouteSettings instead.
+
+## Context
+
+With the introduction of [`Page`][] class,
+the RouteSettings.copyWith was no longer a viable API.
+
+## Description of change
+
+RouteSettings copyWith was removed
+
+## Migration guide
+
+Existing apps:
+
+```dart
+RouteSettings newSettings = oldSettings.copyWith(name: 'new name');
+```
+
+After migrations:
+
+```dart
+RouteSettings newSettings = RouteSettings(name: 'new name', arguments: oldSettings.arguments);
+```
+
+
+## Timeline
+
+Landed in version: 3.5.0-9.0.pre-137-gc6f6095acd<br>
+In stable release: NEXT
```

Comment:
What should i put in here?

Reply:
@sfshaza2 to confirm, but I believe we leave this as TBA until it actually ships in a stable release.

Reply:
changed to TBA